### PR TITLE
zf3 support

### PR DESCRIPTION
**Pull request by kokspflanze**
Hi,

prepare zf3 support

i changed the factory system so zf2 and zf3 should now work.

added php-nightly in travis

known problem:
- doctrine-orm-module dev-master in require-dev
- PHP 5.5 support droped in doctrine-orm-module

my PR goes to develop, so everyone can test the changes if ppl upgrade to zf3 too.

from my site we can also drop PHP 5.5 support (http://php.net/supported-versions.php)

### CODE REVIEW

File: src/ZfcDatagrid/Renderer/TCPDF/Renderer.php [hunk 1]
```diff
@@ -314,10 +313,10 @@ protected function printTableRow(array $row, $rowHeight)
                             if ($file !== false) {
                                 list($width, $height) = $this->calcImageSize($file, $col->getWidth() - 2, $rowHeight - 2);
 
-                                $pdf->Image('@' . $file, $x + 1, $y + 1, $width, $height, '', '', 'L', false);
+                                $pdf->Image('@' . $file, $x + 1, $y + 1, $width, $height, '', '', 'L', true);
```
Comment (ThaDafinser) [inline src/ZfcDatagrid/Renderer/TCPDF/Renderer.php]: Why you need to change the resize param?

File: src/ZfcDatagrid/Renderer/TCPDF/Renderer.php [hunk 2]
```diff
@@ -314,10 +313,10 @@ protected function printTableRow(array $row, $rowHeight)
...
                             }
                         } else {
-                            $pdf->Image($link, $x + 1, $y + 1, 0, $resizeHeight, '', '', 'L', false);
+                            $pdf->Image($link, $x + 1, $y + 1, 0, $resizeHeight, '', '', 'L', true);
```
Comment (ThaDafinser) [inline src/ZfcDatagrid/Renderer/TCPDF/Renderer.php]: Why you need to change the resize param?

**Comment by ThaDafinser** [comment]
@kokspflanze thank you for the PR.

Byside the TCPDF change, why does tag `1.0` from DoctrineORM does not work?
https://github.com/doctrine/DoctrineORMModule/releases

**Comment by ThaDafinser** [comment]
:+1:  for php 5.5 drop. Since it's EOL http://php.net/eol.php

**Comment by kokspflanze** [comment]
doctrine 1.0 doesnt support ZF3-mvc, so for the unit test and all testing parts i added temp the dev-master, we can remove it if they create a new release.

**Comment by kokspflanze** [comment]
the TCPDF changes come from https://github.com/ThaDafinser/ZfcDatagrid/pull/237 that is atm not in develop =)

**Comment by kokspflanze** [comment]
dev-master removed with https://github.com/ThaDafinser/ZfcDatagrid/pull/246

**Pull request merged**